src/main/java/jimmy/mcgymmy/logic/predicate/NameContainsKeywordsPredicate.java

```diff
@@ -18,8 +18,12 @@ public NameContainsKeywordsPredicate(List<String> keywords) {
 
     @Override
     public boolean test(Food food) {
-        return keywords.stream()
-                .anyMatch(keyword -> StringUtil.containsWordIgnoreCase(food.getName().fullName, keyword));
+        try {
+            return keywords.stream()
+                    .anyMatch(keyword -> StringUtil.containsWordIgnoreCase(food.getName().fullName, keyword));
+        } catch (Exception e) {
+            return true;
+        }
```

Comment:
Is there any case where this test can really get an exception?

Reply:
Code didn't work without this line lol, but I am still refactoring, this is just to make it work for now.